Backend/src/Trackable.Web/Controllers/LocationsController.cs

```diff
@@ -119,5 +119,12 @@ public class LocationsController : ControllerBase
 
             return this.dtoMapper.Map<LocationDto>(result);
         }
+
+        // DELETE api/locations/5
+        [HttpDelete("{id}")]
+        public async Task Delete(int id)
+        {
+            await this.locationService.DeleteAsync(id);
```

Comment:
This function should have the return type `Task<IActionResult>`
and should `return Ok();` to indicate successful deletion